src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/UpdateOrgContactInformationIntegrationTest.java

```diff
@@ -0,0 +1,113 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.professionalapi.controller.request.ContactInformationCreationRequest;
+import uk.gov.hmcts.reform.professionalapi.util.AuthorizationEnabledIntegrationTest;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.createContactInformationRequest;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.organisationRequestWithAllFieldsAreUpdated;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.organisationRequestWithMultipleAddressAllFields;
+
+class UpdateOrgContactInformationIntegrationTest extends AuthorizationEnabledIntegrationTest {
+
+    @Test
+    void update_addressline1_for_organisation_with_prd_admin_role_should_return_200() {
+        ContactInformationCreationRequest contactInformationCreationRequest =
+            createContactInformationRequest().build();
+
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(contactInformationCreationRequest,
+                hmctsAdmin,createOrganisationRequest(),true);
+
+        assertThat(updateResponse).containsEntry("http_status", 200);
+    }
+
+    @Test
+    void update_contact_with_bad_request_should_return_400() {
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(null,  hmctsAdmin,createOrganisationRequest(),true);
+
+        assertThat(updateResponse).containsEntry("http_status", "400");
+    }
+
+    @Test
+    void update_contact_information_invalid_addressline1_should_return_400() {
+        ContactInformationCreationRequest contactInformationCreationRequest =
+            createContactInformationRequest().addressLine1(null).build();
+
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(contactInformationCreationRequest,
+                hmctsAdmin,createOrganisationRequest(),true);
+
+        assertThat(updateResponse).containsEntry("http_status", "400");
+        assertThat(updateResponse.get("response_body").toString()).contains(
+            "Field error in object 'contactInformationCreationRequest' on field 'addressLine1'");
+    }
+
+    @Test
+    void update_contact_information_multiple_requests_existing_for_org_should_return_200() {
```

Comment:
test case updating only one address details but method name indicates multiple requests , correct the method name .